packages/tools/src/tools/annotation/LengthTool.ts

```diff
@@ -690,7 +690,7 @@ class LengthTool extends AnnotationTool {
       return;
     }
 
-    const textLines = [`${length.toFixed(2)} ${unit}`];
+    const textLines = [`${length.toFixed(2)} ${unit ?? 'mm'}`];
```

Comment:
Down in _calculateCachedStats, the unit is read - could you add a parent class function to AnnotationTool that gets the unit instead, and has the default unit for the image type?  Something like:
protected getUnit(image,position): string
This should be called for one of the points in the annotation - don't assume the units are consistent across the image, because they aren't for ultrasound (units can be Hertz in one part, mm in another, and mm/s in a third part).  Then, that call can be used in cached stats, and for the default cached stats, you can just assign the default image units, or reference it via an attribute, eg:
this.defaultUnit

That also allows fixing the units to show projected versus imaging units when relevant for CR/DX images.

Reply:
Hmmm, I hear what you are saying here, but for ultrasound, are you saying the pixelSpacing can be anything other than mm or px? Isn't the unit for pixelSpacing mm in dicom?

Reply:
Probably don't need to worry about different units right now, but having a function called in the parent would be useful, as there are other bits that are sometimes important to check.

Reply:
I ended up don't changing/assigning default units for this PR, can you review the latest?

Reply:
Yes, the latest looks good.